front/app/containers/IdeasShowPage/IdeaShowPageTopBar.tsx

```diff
@@ -138,18 +142,27 @@ const IdeaShowPageTopBar = ({
           />
         </Left>
         <Right>
-          {/* Only visible if not voting */}
-          {phase?.attributes.participation_method !== 'voting' && ( // To reduce bias we want to hide the reactions during voting methods
```

Comment:
This was the bug. The Vote button would always show on mobile. The conditions from the desktop view were not applied here.